core/common/src/main/java/alluxio/collections/LockCache.java

```diff
@@ -267,14 +219,12 @@ public int getSoftLimit() {
   /**
    * Node containing value and other information to be stored in the cache.
    */
-  public class ValNode {
-    private ReentrantReadWriteLock mValue;
-    private boolean mIsAccessed;
```

Comment:
there is no more clock approximation?

Reply:
I think the reference counter is enough, what's the point of this boolean?